### DISCUSSION TITLE
What's the preferred way to report a security vulnerability regarding Cider?

### DISCUSSION THREAD

**Discussion by maple3142** [ciderapp/Cider#1689] [General]
There is no `SECURITY.md`, and it is not possible to [privately reporting a security vulnerability](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability) either.

**Comment by cunev** [comment]
This just proves that this project completely doesn't care about security.
Ignore and use at your own risk vibe.

**Comment by GamingLiamStudios** [comment]
Please use our [Discord](https://discord.gg/applemusic) ticket channel to report any Cider related security vulnerabilities. Do note however that Cider1 (the open-source one) is EoL and may not have any security patches in the future.
(yes I know this is several months old but for future reference in case anyone else finds something spicy)
